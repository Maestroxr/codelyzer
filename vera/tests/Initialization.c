

int main() 
{
	/*OK*/
	int a1 = 1;
	int a2 = 1, a3 = 1;
	/*********/	

	/*PROBLEM*/
	int problem1;
	int problem2, problem3;
	int* problem4, b = 2; 
	int c = 2, problem5; 
	char** problem6, s2 = 2, problem7;
	char s4 = 2,problem8 , s6 = 2;
	/*********/	
	
	return 0;
	
}


