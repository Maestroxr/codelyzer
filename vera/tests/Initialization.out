C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:11: Uninitialized identifier 'problem1' of type int.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:12: Uninitialized identifier 'problem3' of type int.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:12: Uninitialized identifier 'problem2' of type int.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:13: Uninitialized identifier 'problem4' of type int*.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:14: Uninitialized identifier 'problem5' of type int.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:15: Uninitialized identifier 'problem7' of type char.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:15: Uninitialized identifier 'problem6' of type char**.
C:\Work\AutoAssessment\codelyzer\vera\tests\Initialization.c:16: Uninitialized identifier 'problem8' of type char.
